ILSwiss/rlkit/torch/common/networks.py

```diff
@@ -101,6 +97,184 @@ def forward(self, input, return_preactivations=False):
         else:
             return output
 
+    @torch.jit.export
+    def jit_forward(self, input):
+        assert self.layer_norm is False
+        assert self.batch_norm is False
+        assert self.batch_norm_before_output_activation is False
+        h = input
+        for i, fc in enumerate(self.fcs):
+            h = fc(h)
+            h = self.hidden_activation(h)
+        preactivation = self.last_fc(h)
+        output = self.output_activation(preactivation)
+        return output
```

Comment:
Just curious to ask, how much acceleration does jit roughly provide?
and what is the minimal version of PyTorch required to use jit?

Reply:
`jit forward` is not used for training. It is used only for saving an script model for C++ inference.